Obsidian.API/_Types/Painting.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Obsidian.API
+{
+    public class Painting
+    {
+
+        public readonly short Id;
+        public readonly byte X;
+        public readonly byte Y;
+        public readonly byte Width;
+        public readonly byte Height;
```

Comment:
These could be properties.

_Since it's not a lot of data, `Painting` could even be a struct?_

Reply:
+1 on that